lightkurve/tests/test_periodogram.py

```diff
@@ -22,8 +23,10 @@ def test_periodogram_basics():
                     flux_err=np.zeros(1000)+0.1)
     lc = lc.normalize()
     pg = lc.to_periodogram()
-    pg.plot()
-    pg.plot(view='period')
+    ax = pg.plot()
+    plt.close()
+    ax = pg.plot(view='period')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'ax' is assigned to but never used](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4938792)